spec/features/sdr_deposit_spec.rb

```diff
@@ -56,26 +56,64 @@
     # Wait for accessioningWF to finish
     reload_page_until_timeout!(text: 'v1 Accessioned')
 
-    # Download Gemfile (preserved=true) from Preservation
-    click_link_or_button 'Gemfile'
-    expect(page).to have_text 'Preservation:'
-    gemfile_pres_link = find('.modal-content a')
-    gemfile_pres_url = gemfile_pres_link['href']
-    expect(gemfile_pres_url.end_with?("/items/#{druid}/files/Gemfile/preserved?version=1")).to be true
-
-    click_link_or_button gemfile_pres_link.text
+    retries_count = 0
```

Comment:
Can you put in a comment about why you're retrying?

Reply:
i did put one at the top of the `rescue` block, but i'm happy to move it up here if you think that's a better location (i think maybe it is, now that you've pointed this out)